source/read/distinct.txt

```diff
@@ -0,0 +1,152 @@
+value in the ``borough`` field, each value appears in the results only once.
+
+Retrieve Distinct Values Across Specified Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can provide a **query filter** to the ``distinct`` command to find the distinct
+field values across a subset of documents in a collection. A query filter is an expression that specifies search
+criteria used to match documents in an operation.
+
+.. TODO
+.. For more information about
+.. creating a query filter, see :ref:`c-specify-query`.
+
+The following example retrieves the distinct values of the ``borough`` field for
+all documents that have a ``cuisine`` field value of ``"Italian"``:
+
+.. io-code-block::
+
+   .. input:: /includes/read/distinct.c
+      :start-after: start-distinct-query
+      :end-before: end-distinct-query
+      :language: c
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      { "values" : [ "Bronx", "Brooklyn", "Manhattan", "Queens", "Staten Island" ], ... }
+
+Modify Distinct Behavior
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``distinct`` command can be modified by passing options to the ``mongoc_collection_read_command_with_opts()``
+function. If you don't specify any options, the driver does not customize the operation.
+
+The following table describes some methods you can use to customize the
+``distinct`` operation:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``comment``
+     - | Specifies a comment to attach to the operation.
+
+For a complete list of methods you can use to modify the ``distinct`` method, see
+the :manual:`distinct command </reference/command/distinct>` documentation in the
+{+mdb-server+} manual.
+
+The following example retrieves the distinct values of the ``name`` field for
+all documents that have a ``borough`` field value of ``"Bronx"`` and a
+``cuisine`` field value of ``"Pizza"``. It also uses
+the ``comment`` option to add a comment to the operation.
+
+.. io-code-block::
+
+   .. input:: /includes/read/distinct.c
+      :start-after: start-distinct-comment
+      :end-before: end-distinct-comment
+      :language: c
+      :dedent:
+
+   .. output:: 
+      :visible: false
+
+      { "values" : [ "$1.25 Pizza", "18 East Gunhill Pizza", "2 Bros", "Aenos Pizza", "Alitalia Pizza Restaurant", ... ], ... }
+
+Additional Information
+----------------------
+
+To learn more about the distinct command, see the :manual:`Distinct guide
+</reference/command/distinct/>` in the MongoDB Server Manual.
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+To learn more about any of the methods or types discussed in this
+guide, see the following API documentation:
```

Comment:
S: since there's only one method listed here, might make sense to reword as:
```suggestion
To learn more about the ``mongoc_collection_read_command_with_opts()`` method, see the `API documentation <{+api-libmongoc+}/mongoc_collection_read_command_with_opts.html>`__.
```